src/codeu/chat/server/Server.java

```diff
@@ -176,7 +190,8 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
     this.commands.put(NetworkCode.GET_CONVERSATIONS_BY_ID_REQUEST, new Command() {
       @Override
       public void onMessage(InputStream in, OutputStream out) throws IOException {
-
+        final Uuid userId = Uuid.SERIALIZER.read(in);
```

Comment:
The user id was added to create a user context for security, but since it doesn't map 1:1, the line below was commented out so that I could figure out how to make this secure.